RetroBar/Languages/English.xaml

```diff
@@ -20,6 +20,15 @@
         <s:String>Right</s:String>
         <s:String>Bottom</s:String>
     </x:Array>
+    <s:String x:Key="rowcount_text">Number of _rows:</s:String>
+    <s:String x:Key="rowcount_tip">The number of rows on the taskbar when on top or bottom.</s:String>
+    <x:Array x:Key="rowcount_options" Type="s:Int32">
+        <s:Int32>1</s:Int32>
+        <s:Int32>2</s:Int32>
+        <s:Int32>3</s:Int32>
+        <s:Int32>4</s:Int32>
+        <s:Int32>5</s:Int32>
+    </x:Array>
```

Comment:
This is not localizable, it shouldn't be here.

Reply:
I didn't mean it like that, but that this numeric value is misused as a configuration, and that should be done behind code, not in the resource file for translation.

Reply:
Oh oops. That's left over from just getting it working, not gonna lie

Reply:
@xoascf Good catch! I will get this fixed.